glusterd2/peer/self.go

```diff
@@ -46,7 +46,8 @@ func AddSelfDetails() error {
 		Name:          gdctx.HostName,
 		PeerAddresses: []string{config.GetString("peeraddress")},
 	}
-
+	p.MetaData = make(map[string]string)
+	p.MetaData["group"] = "1"
```

Comment:
The value `1` seems very arbitrary. You don't need to set a default group at all. Unless metadata is explicitly is set by the user (or a plugin/program), it shouldn't exist.